docs/content/guides/getting-started/binding-to-data.md

```diff
@@ -54,43 +54,38 @@ const hot = new Handsontable(container, {
 ::: only-for react
 ::: example #example1 :react
 ```jsx
-import React, { Fragment, useEffect } from 'react';
+import React from 'react';
```

Comment:
Is this import needed? I don't see `React` being used anywhere in the snippet.

Reply:
i guess not if fiddle supports it